Tests/SwiftWyhashTests/SwiftWyhashTests.swift

```diff
@@ -8,15 +8,33 @@ final class SwiftWyhashTests: XCTestCase {
         XCTAssertEqual(gen.next(), 13322404298164966600)
         XCTAssertEqual(gen.next(), 10710867605997789043)
     }
-    
+
     func testShuffled() {
         var g = Wyhash64(seed: 0)
         XCTAssertEqual((1...10).shuffled(using: &g),
                        [1, 6, 2, 8, 4, 7, 5, 10, 3, 9])
     }
 
+    func testNumberSequence() {
+        var numbers = sequence(state: Wyhash64(seed: 0)) {
+            (g: inout Wyhash64) -> Int? in
+            return Int.random(in: Int.min...Int.max, using: &g)
```

Comment:
This could have been written more succinctly using implicit closure parameter as:
```
var numbers = sequence(state: Wyhash64(seed: 0)) {
    return Int.random(in: Int.min...Int.max, using: &$0)
}
```